docs/src/man/getting_started.md

```diff
@@ -15,19 +15,27 @@ relevant variables into your current namespace.
 
 !!! note
 
-    By default Jupyter Notebook will limit the number of rows and columns when displaying a data frame to roughly
-    fit the screen size (like in the REPL).
-
-    You can override this behavior by changing the values of the `ENV["COLUMNS"]` and `ENV["LINES"]`
-    variables to hold the maximum width and height of output in characters respectively.
+    By default DataFrames.jl limits the number of rows and columns when displaying a data frame in a Jupyter
+    Notebook to 25 and 100, respectively. You can override this behavior by changing the values of the
+    `ENV["DATAFRAMES_COLUMNS"]` and `ENV["DATAFRAMES_ROWS"]` variables to hold the maximum number of columns
```

Comment:
I understand that setting them for a value of 0 or less means print all? If this is the case could you please add this as a comment? Thank you!